components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -455,21 +473,39 @@ def make_config_paths_absolute(self, clp_home: pathlib.Path):
 class FsIngestionConfig(FsStorage):
     directory: SerializablePath = pathlib.Path("/")
 
+    def transform_for_container(self):
+        input_logs_dir = self.directory.resolve()
+        self.directory = CONTAINER_INPUT_LOGS_ROOT_DIR / input_logs_dir.relative_to(
+            input_logs_dir.anchor
+        )
+
 
 class ArchiveFsStorage(FsStorage):
-    directory: SerializablePath = CLP_DEFAULT_DATA_DIRECTORY_PATH / "archives"
+    directory: SerializablePath = CLP_DEFAULT_ARCHIVES_DIRECTORY_PATH
+
+    def transform_for_container(self):
```

Comment:
The "native" scripts - particularly n`ative/dataset_manager.py` - should have been updated to use the new in-container path (`/var/data`) instead of relying on the previous direct host-to-container path mapping. Otherwise, the dataset manager isn't able to remove archives at the moment.

@coderabbitai create an issue to track the archive deletion bug in the dataset manager

and another one to migrate the sbin tools to Docker Compose - or at least update their Docker mappings to align with the Package Compose project